README.md

```diff
@@ -1 +1,61 @@
-# Advance_todo
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Advance_todo
+> A simple Todo app to add list of books a user has read. It is implemented using HTML, CSS, Webpack, JavaScript and modular architecture. A todo is saved and displayed in Realtime. A todo can be removed from the list of to-dos. Data persistence is implemented using local Storage API.
+
+![screenshot](./screenshot.JPG)
+
```

Comment:
[OPTIONAL]

- The `screenshot` is breaking. Please use the correct image to show in the README to make professional.